CONTRIBUTING.md

```diff
@@ -98,7 +112,15 @@ Now he wants to go back to fixing his weather plugin, he changes branch
 git checkout weather-plugin
 ```
 
-works, then commit
+test if all files are well formatted, complying with style and security rules, before send the PR
+
+```
+black --check --verbose --config ./pyproject.toml src/honeybot/plugins/downloaded/weather/main.py
+isort --check-only --settings-path ./pyproject.toml src/honeybot/plugins/downloaded/weather/main.py
+bandit -ll -c ./pyproject.toml -r src/honeybot/plugins/downloaded/weather/main.py 
+```
```

Comment:
Should we recommend checking parts on which changes were made only or the whole project as the default?

I'm worried new contributors may get confused by their checks passing but the PR branch checks failing because of `black`, `isort`, `bandit` failing on the entire codebase.

Reply:
@MrTanoshii Well, pre-commit only checks for the changes. We should change the CI @paulosgf to enforce only what is required by pre-commit. This is industry-practice.

Reply:
Sure thing then, we'll keep it as is

Reply:
> @MrTanoshii Well, pre-commit only checks for the changes. We should change the CI @paulosgf to enforce only what is required by pre-commit. This is industry-practice.

Sure. I would like to send a new plugin that I've been working first. 
Then, I update this, Ok?